src/features/chat/ui/chatRoomList/ChatRoomItem.tsx

```
@@ -0,0 +1,27 @@
+export default function ChatRoomItem({ room, isSelected }: IChatRoomItemProps) {
+  const { roomId, title, headCount } = room;
+  const { handleSwitchRoom } = useChatDialogTrigger();
+
+  return (
+    <li
+      className={clsx(
+        'p-3 rounded-[10px] cursor-pointer transition-all duration-200 w-full  text-white',
+        isSelected ? 'bg-primary' : 'hover:text-secondary hover:bg-white/8 '
+      )}
+      onClick={() => handleSwitchRoom(roomId, title)}
+    >
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)